assets/src/features/shared/utils/isMobileDevice.tsx

```diff
@@ -0,0 +1 @@
+export const isMobileDevice = /iPhone|iPad|iPod|Android|webOS|BlackBerry|Windows Phone/i.test(navigator.userAgent);
```

Comment:
I don't think it's the best idea to use `navigator.userAgent` 😿 

** (...) you really have no guarantee that the browser agent is indeed the one advertised by this property.

Also keep in mind that users of a browser can change the value of this field if they want (UA spoofing).**

Source: [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgent)


Maybe [`window.matchMedia()`](https://stackdiary.com/detect-mobile-browser-javascript/#window-matchmedia) is a better solution?

Reply:
> Browser support for this standard is ~96%

That's more than enough, our WebRTC support is more limited than that and any reasonable browser seems to support this standard.

> laptops < 13’ with touch screens

My concern exactly. Wouldn't it be better to use `pointer` instead of `any-pointer`? I think it has a better chance of working properly on such a laptop, but I don't think we have such a device to test it

Reply:
> tablets with external keyboards

Both scenarios are quite unlikely, but IMO tablets with a mouse are less common. Plus, it doesn't totally break but shows a button that isn't working which is less of a problem than not showing this button to a user that could use it